src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx

```diff
@@ -0,0 +1,27 @@
+---
+subject: PHP agent
+releaseDate: '2022-03-02'
+version: 9.19.0.309
+downloadLink: 'https://download.newrelic.com/php_agent/archive/9.19.0.309'
```

Comment:
Change to:
getStartedLink: '....'

Reply:
Using downloadLink is consistent with release notes for previous releases of the agent. Please take a look at previous two releases: [v9.17.1](https://github.com/newrelic/docs-website/blob/develop/src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9171301.mdx) and [v9.18.0](https://github.com/newrelic/docs-website/blob/develop/src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9181303.mdx). Please see on this screenshot to the right how it renders:
<img width="570" alt="image" src="https://user-images.githubusercontent.com/98839644/156463632-62190172-79a3-45c4-a760-6c36c15dc0c0.png">